maryana_khoptyak/Task_1/test_get_max_value.py

```diff
@@ -0,0 +1,44 @@
+EMPTY_SEQUENCE = []
```

Comment:
It's much better to have constant variables in alphabetic order.

Reply:
Constant variables are sorted in alphabetic order.